rust/src/mask/masking.rs

```diff
@@ -128,6 +172,25 @@ impl Aggregation {
             .collect()
     }
 
+    /// Validates if aggregation of the aggregated mask or masked model with the given `object`
```

Comment:
As said in another comment, maybe we could talk about "mask object" here.

Reply:
i updated [the other comment](https://github.com/xainag/xain-fl/pull/435#discussion_r448936330) accordingly.